packages/neovim-nightly/src-nvim-os-stdpaths.c.patch

```diff
@@ -0,0 +1,14 @@
+diff -u -r ../neovim-master/src/nvim/os/stdpaths.c ./src/nvim/os/stdpaths.c
+--- ../neovim-master/src/nvim/os/stdpaths.c	2015-11-01 16:16:38.000000000 -0500
++++ ./src/nvim/os/stdpaths.c	2015-11-02 14:12:12.770172673 -0500
+@@ -34,8 +34,8 @@
+   [kXDGDataHome] = "~/.local/share",
+   [kXDGCacheHome] = "~/.cache",
+   [kXDGRuntimeDir] = NULL,
+-  [kXDGConfigDirs] = "/etc/xdg/",
+-  [kXDGDataDirs] = "/usr/local/share/:/usr/share/",
++  [kXDGConfigDirs] = "@TERMUX_PREFIX@/etc/xdg/",
++  [kXDGDataDirs] = "@TERMUX_PREFIX@/local/share/:@TERMUX_PREFIX@/share/",
```

Comment:
Shouldn't this be `@TERMUX_HOME@/.local/share/:@TERMUX_PREFIX@/share/` ?

Reply:
My bad the same patch is for neovim, so resolving this also it seems to respect `~/.local/share`